knative-operator/pkg/controller/knativeeventing/knativeeventing_controller_test.go

```diff
@@ -45,105 +42,3 @@ func init() {
 
 	apis.AddToScheme(scheme.Scheme)
 }
-
-// TestEventingReconcile runs Reconcile to verify if eventing resources are created/deleted.
-func TestEventingReconcile(t *testing.T) {
```

Comment:
I converted this test to an E2E test, see VerifyDashboards.

Reply:
+1